rasa_nlu/extractors/crf_entity_extractor.py

```diff
@@ -42,28 +47,32 @@ class CRFEntityExtractor(EntityExtractor):
         # features to use for each word, for example, 'title' in
         # array before will have the feature
         # "is the preceding word in title case?"
+        # POS features require spaCy to be installed
         "features": [
-            ["low", "title", "upper", "pos", "pos2"],
-            ["bias", "low", "word3", "word2", "upper",
-             "title", "digit", "pos", "pos2", "pattern"],
-            ["low", "title", "upper", "pos", "pos2"]],
+            ["low", "title", "upper"],
+            ["bias", "low", "prefix5", "prefix2", "suffix5", "suffix3",
+             "suffix2", "upper", "title", "digit", "pattern"],
+            ["low", "title", "upper"]],
 
         # The maximum number of iterations for optimization algorithms.
         "max_iterations": 50,
 
         # weight of theL1 regularization
-        "L1_c": 1,
+        "L1_c": 0.1,
 
         # weight of the L2 regularization
-        "L2_c": 1e-3
+        "L2_c": 0.1
     }
 
     function_dict = {
         'low': lambda doc: doc[0].lower(),
         'title': lambda doc: doc[0].istitle(),
-        'word3': lambda doc: doc[0][-3:],
```

Comment:
this makes the model backwards incompatible. In `Interpreter` `L255` there is a model backwards compatibility check that you should increase then.